### DISCUSSION TITLE
look for duplicate hashes, on different hard drives

### DISCUSSION THREAD

**Discussion by paulobreim** [sepinf-inc/IPED#1442] [Ideas]
I would like to propose a suggestion.

Motivation: We are seeing more and more cases of theft of information within the company, committed by former employees. They use pendrive to copy information, send e-mail, use bluetooth, etc.
Usually this ends up in a search and seizure process at the person's home or at the competing company when he saw work there.

Expertise: After the search and seizure, the expert's job is to look for information on the HD belonging to the company that started the process. The comparison often involves the content of the HD that the employee used in the company compared to the HD that was seized.

Solution: Index the 2 HDs through IPED.
Create a button or command in which all duplicate files are shown as long as they are present on different HDs.
I don't think any other app of this type has this functionality.

**Comment by lfcnassif** [comment]
I think you can already achieve a similar result currently:
1. Check all items of the first HD using the UI
2. Bookmark all those checked items enabling the "include duplicates" feature in bookmark manager dialog
3. Then all unchecked items in the new bookmark will be duplicates present in the second HD

You can swap the first/second HDs and reapply steps above to find all duplicates in the other direction.
